tests/data/group_services/group_nfs_and_rpc/group_nfs_configuring_clients/group_mounting_remote_filesystems/fstab

```diff
@@ -0,0 +1,6 @@
+# /etc/fstab: static file system information
+# Example fstab file with mounted NFS file systems
+# <file system>                   <mount point>   <type>  <options>                             <dump>  <pass>
+
+some.server:/remote/folder1         /folder1       nfs    defaults,noexec,nosuid,sec=krb5:krb5i:krb5p     0  0
+another.server:/remote/folder2      /folder2       nfs4   defaults,noexec,nosuid,sec=krb5:krb5i:krb5p     0  0
```

Comment:
There could be other unrelated mounts here besides the `nfs` ones. Or it could another test scenario, not sure if it is needed to have two scenarios for pass scenario in which all mounts of `nfs` type have the kerberos options.